ddtrace/contrib/dd_trace_api/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""
+
+Enabling
```

Comment:
I don't think we need this file, this isn't really something that needs to be documented, right?

no one should ever interact with this manually?

Reply:
ah, true